tests/cases/fourslash/tripleSlashRefPathCompletionExtensionsAllowJSFalse.ts

```diff
@@ -0,0 +1,31 @@
+/// <reference path='fourslash.ts' />
+
+// Should give completions for relative references to ts files when allowJs is false
```

Comment:
I see lots of tests where `allowJs` is `false`. Is there sufficient coverage for checking that when it's `true` the `.js[x]` files do appear in references and imports?